lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -116,9 +116,7 @@ define([
         showErrorMessage: function (message) {
             this.$('.wrapper-msg')
                 .removeClass('is-hidden')
-                .find('.msg-content .copy').text(message)
-                .closest('.msg')
-                .focus();
```

Comment:
> I tried to adjust focus to the entire .wrapper-msg element using both the .parent() and .closest() JQuery methods, but it seemed not to work.

We should add `tabindex="-1"` to the element allows it to receive programmatic focus. So, `.msg-content` should looks so:
```html
<div class="msg-content" tabindex="-1">
  <p class="copy"></p>
</div>
```
Also, we can simply apply `outline: none;` to hide the border on focus.